tests/seeding/test_find_or_create.py

```diff
@@ -67,6 +70,20 @@ def _make_project(search_implemented: bool = True):
     return _make_project
 
 
+@pytest.fixture
+def team_collection() -> Callable[[bool], TeamCollection]:
+
+    def _make_team(search_implemented: bool = True):
```

Comment:
In each case you use this fixture, you don't pass any arguments on invocation. Any reason to not flatten it out?